history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
```

Comment:
I would have thought that this is too minor a bug-fix to mention in release announcement? 
But if we want to include it, perhaps: "Fixed channel buttons turning black in OMERO.web" and put it further down the list?